bazel/defaults.bzl

```diff
@@ -0,0 +1,21 @@
+load("@npm//@bazel/concatjs:index.bzl", _ts_library = "ts_library")
```

Comment:
Technically we could've fixed the issue by adding `prodmode_target` and `devmode_target` to all the places using `ts_library`, but I figured that this is a bit more maintainable.